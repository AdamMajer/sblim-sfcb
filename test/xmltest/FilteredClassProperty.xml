<?xml version="1.0" encoding="UTF-8"?>
<!-- From sfcb-tix#24 -->
<CIM CIMVERSION="2.0" DTDVERSION="2.0"><MESSAGE ID="1000" PROTOCOLVERSION="1.0">
<SIMPLEREQ><IMETHODCALL NAME="GetClass">
<LOCALNAMESPACEPATH><NAMESPACE NAME="root"/><NAMESPACE NAME="cimv2"/></LOCALNAMESPACEPATH>
 <IPARAMVALUE NAME="ClassName"><CLASSNAME NAME="Linux_Processor"/></IPARAMVALUE>
 <IPARAMVALUE NAME="LocalOnly"><VALUE>FALSE</VALUE></IPARAMVALUE>
 <IPARAMVALUE NAME="PropertyList">
  <VALUE.ARRAY><VALUE>NumberOfEnabledCores</VALUE></VALUE.ARRAY>
 </IPARAMVALUE><IPARAMVALUE NAME="IncludeQualifiers"><VALUE>FALSE</VALUE></IPARAMVALUE></IMETHODCALL></SIMPLEREQ></MESSAGE></CIM>
