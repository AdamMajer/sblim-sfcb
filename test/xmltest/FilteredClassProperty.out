500 Can't connect to localhost:5988 (Connection refused)
